src/client/components/equalizer/upload/infoAboutFile/infoAboutUploadFile.css

```diff
@@ -0,0 +1,4 @@
+#detailstrack {
+  margin-top: 15px;
+  text-align: center;
+}
```

Comment:
missed empty line at the end of the file.

Reply:
done)